source/dagger/gameplay/team_game/gravity.cpp

```diff
@@ -0,0 +1,32 @@
+#include "gravity.h"
+
+#include "core/core.h"
+#include "core/engine.h"
+#include "core/game/transforms.h"
+
+#include "gameplay/team_game/team_game_collisions.h"
+
+using namespace dagger;
+
+void team_game::GravitySystem::Run()
+{
+    Engine::Registry().view<Collider, Gravity, Transform>().each([&](Collider& collider_, Gravity& gravity_, Transform& transform_) 
+        {
+            if (collider_.canGoDown)
+            {                
+                if (gravity_.terminalVelocity < m_GravityStrength * gravity_.airTime)
+                {
+                    transform_.position.y -= gravity_.terminalVelocity;
```

Comment:
You are subtracting velocity [m/s] from a coordinate [m]. I think it's a good idea to change this into gravity_.terminalVelocity * gravity_.airTime, and also the line at 23 to m_GravityStrength * gravity_.airTime * gravity_.airTime and then tweak the JumpSpeed accordingly. It's more realistic that way in my opinion.

Reply:
Hmm, yeah, that does not seem logical. Also it just occured to me that the actual terminal velocity is framerate dependent which is going to have to be changed

Reply:
As for this request, I tried making changes requested but i feel it compromises the feel of gravity, which should be linear and not quadratic since i view Transforms to be changes in position over some amount of time(in this case Engine::DeltaTime())